tests/integration/offer/test_range.py

```diff
@@ -231,3 +232,52 @@ def test_ensures_unique_slugs_are_used(self):
         first_range.name = "Bar"
         first_range.save()
         models.Range.objects.create(name="Foo")
+
+
+class TestRangeQuerySet(TestCase):
```

Comment:
@specialunderwear all these tests are passing for me without any change to `apps/offer/queryset.py`? 

I think it would be helpful to add a test that fails before applying the change so that we (a) know what has been fixed and (b) ensure it doesn't regress in future.

Also there's a couple of typos in the strings below "shoudl" -> "should"